examples/Demo/Shared/Pages/SplashScreen/Examples/CustomSplashScreen.razor

```diff
@@ -33,7 +37,7 @@
         if (firstRender)
         {
             // Simulation of loading process
-            await Task.Delay(7000);
+            await Task.Delay(70000);
```

Comment:
Is it normal to increase the time to 70 seconds?

Reply:
Oops, that was for testing only. Corrected it